hi-ml-cpath/src/health_cpath/datasets/tcga_prad_dataset.py

```diff
@@ -37,6 +37,5 @@ def __init__(self, root: Union[str, Path],
         """
         super().__init__(root, dataset_csv, dataset_df, validate_columns=False)
         # Example of how to define a custom label column from existing columns:
-        self.dataset_df[self.LABEL_COLUMN] = (self.dataset_df['label1']
-                                              | self.dataset_df['label2']).astype(int)  # noqa: W503
+        self.dataset_df[self.LABEL_COLUMN] = (self.dataset_df['brca1_mutation']).astype(int)  # noqa: W503
```

Comment:
Reason for changing this here? We will be making all TCGA related changes in the internal repository.

Reply:
Needed the change to check if the code runs on TCGA. I am aware that you are currently changing/cleaning this part.